portfolio/src/main/webapp/style.css

```diff
@@ -196,6 +273,42 @@ p {
   margin: 30px auto;
   outline: none;
   padding: 10px;
+  transition: filter var(--transition-speed);
   text-transform: uppercase;
   width: fit-content;
+}
+
+.btn:hover {
+  cursor: pointer;
+  filter: brightness(85%);
+
+}
+
+#back-to-top-btn {
```

Comment:
This is a small design comment, but I feel like the back-to-top button could be larger or easier to see. I missed it the first couple times I scrolled through your site.